build.gradle

```diff
@@ -1,3 +1,7 @@
+plugins {
+    id 'net.researchgate.release' version '2.8.1'
+}
+
```

Comment:
bellow here, on the line 13 we still have the property version, which is a different one from the properties file. Can we remove it?